recipes/enet/all/patches/fix-cmake.patch

```diff
@@ -0,0 +1,20 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -84,11 +84,15 @@ set(SOURCE_FILES
+ source_group(include FILES ${INCLUDE_FILES})
+ source_group(source FILES ${SOURCE_FILES})
+ 
+-add_library(enet STATIC
++if(MSVC AND BUILD_SHARED_LIBS)
```

Comment:
```suggestion
if(WIN32 AND BUILD_SHARED_LIBS)
```

Reply:
I'm able to build and consume enet with MinGW without this modification, symbols are not hidden.